cerberus/tests/tests.py

```diff
@@ -1438,6 +1438,45 @@ def test_coercion_of_sequence_items(self):
         for x in result['a_list']:
             self.assertIsInstance(x, float)
 
+    def test_default_missing(self):
+        schema = {'foo': {'type': 'string'},
+                  'bar': {'type': 'string',
+                          'default': 'bar_value'}}
+
+        self.assertSuccess({'foo': 'foo_value'}, schema)
```

Comment:
you may implement a test-method to test the results of `Validator.normalized`. there should be some tests that can use it. and more will grow.